Sources/Orbit/Orbit.docc/Components/Components.md

```diff
@@ -56,42 +60,47 @@ Our components are a collection of interface elements that can be reused across
 - ``Layout``
 - ``HorizontalScroll``
 
-### Navigation
-
-- ``Breadcrumbs``
-- ``LinkList``
-- ``NavigationBar``
-- ``Pagination``
+[//]: # (### Navigation)
+[//]: # (Breadcrumbs)
+[//]: # (LinkList)
```

Comment:
What is this new syntax?

Reply:
This is the generic markdown comment format - to remove warnings about unused types.

Reply:
TIL